.circleci/config.yml

```diff
@@ -96,20 +96,45 @@ jobs:
           POSTGRES_DB: postgres_test
           POSTGRES_PASSWORD: '12345678'
           POSTGRES_USER: postgres
+    parallelism: 20
```

Comment:
This makes CI expenses 20 times more expensive in exchange of  4x faster builds. I'd tune it to allow parallelism of 2,4 or 4 to optimize both values. Did you test other concurrency values?

Reply:
according to [this report](https://app.circleci.com/insights/github/decentraland/catalyst/workflows/build-deploy/jobs?branch=chore%2Fjest-circleci-with-parallelism), our slowest test time is **30s**.

it is currently splitting by names which is not the idea, if we find out how to split by timings I guess we can use lower values of parallelism and get the same result.

I will test with those values

Reply:
parallelism of 4 and splitting by timings: 2.24s

see [workflow](https://app.circleci.com/pipelines/github/decentraland/catalyst/4707/workflows/1b93b4ef-3688-4d43-92f4-88d409647aff)

credits usage still low compared to `main` branch